torch/_dynamo/variables/builtin.py

```diff
@@ -581,7 +581,28 @@ def call_function(
             )
         return super().call_function(tx, args, kwargs)
 
-    def _call_min_max(self, tx, a, b):
+    def _call_min_max(self, tx, *args):
+        if len(args) == 1 and args[0].has_unpack_var_sequence(tx):
+            return self._call_min_max_iterable(tx, args[0])
+        elif len(args) == 2:
+            return self._call_min_max_binary(tx, args[0], args[1])
+        else:
+            unimplemented(
+                f"unsupported min / max over {len(args)} args: {[str(a) for a in args]}"
+            )
+
+    def _call_min_max_iterable(self, tx, obj):
+        items = obj.unpack_var_sequence(tx)
```

Comment:
assert has_unpack_var_sequence - I know its checked above, but its good to protect the actual callsite :) it raises NotImplementedError if its not impelemented

Reply:
idk, seems kind of pointless to me, if you've got it wrong unpack_var_sequence will blow up anyway, you aren't getting any safety from an extra assert here